vendor/assets/javascripts/leaflet-beautify-marker/leaflet-beautify-marker-icon.js

```diff
@@ -0,0 +1,206 @@
+/*
+  Leaflet.BeautifyIcon, a plugin that adds colorful iconic markers for Leaflet by giving full control of style to end user, It has also ability to adjust font awesome
+  and glyphicon icons
+  (c) 2016-2017, Muhammad Arslan Sajid
```

Comment:
What is the license here, and is it GPL compatible?

Reply:
MIT https://github.com/Leaflet/Leaflet.markercluster/blob/master/MIT-LICENCE.txt